v1/summary.js

```diff
@@ -15,6 +15,9 @@ module.exports = function(options, templates) {
                     thumb.source = thumb.source.replace(/^http:/, 'https:');
                 }
                 return thumb;
+            },
+            getTimestamp: function(items) {
```

Comment:
Done, but I've added the 'getRevision' function returning an object directly instead, looks nicer imho, and we don't need generality here since it's limited to summary endpoint config